inputfiles/overridingTypes.jsonc

```diff
@@ -218,6 +218,38 @@
                         }
                     }
                 }
+            },
+            "WindowOrWorkerGlobalScope": {
+                "methods": {
+                    "method": {
+                        "clearInterval": {
+                            "signature": {
+                                "0": {
+                                    "param": [
+                                        {
+                                            "name": "id",
+                                            "optional": "false",
+                                            "overrideType": "number | undefined"
+                                        }
+                                    ]
+                                }
+                            }
+                        },
+                        "clearTimeout": {
+                            "signature": {
+                                "0": {
+                                    "param": [
+                                        {
+                                            "name": "id",
+                                            "optional": "false",
+                                            "overrideType": "number | undefined"
```

Comment:
`overrideType` should be also a last resort, perhaps `additionalTypes: ["undefined"]`?